src/views/Buy/Buy.js

```diff
@@ -0,0 +1,159 @@
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Grid,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import Header from "components/Header/Header";
+import React, { useEffect, useState } from "react";
+import { database } from "../../firebase/index";
+import "./Buy.css";
```

Comment:
Change it to `./Buy.scss`
`css.map` is not working, infact it doesn't worked on my machine too.